app/src/main/java/edu/wpi/first/shuffleboard/app/Shuffleboard.java

```diff
@@ -34,7 +37,32 @@ public void init() throws AlreadyLockedException, IOException {
       throw alreadyLockedException;
     }
 
-    NetworkTablesJNI.startClient("localhost", 1735);
+    NetworkTable.setClientMode();
+
+    ChangeListener<String> serverChangeListener = (observable, oldValue, newValue) -> {
+      if (newValue.matches("[1-9](\\d{1,3})?")) {
+        NetworkTable.setTeam(Integer.parseInt(newValue));
+      } else if (newValue.isEmpty()) {
+        NetworkTable.setIPAddress("localhost");
+      } else {
+        NetworkTable.setIPAddress(newValue);
```

Comment:
Disagree.  A single field can support either a hostname or a team number.  The problem with having 2 fields is you can only use one at a time.